pkg/cmd/gc/pods/pods_test.go

```diff
@@ -0,0 +1,61 @@
+package pods
```

Comment:
`pods_test` for package name.

Reply:
Same as https://github.com/jenkins-x-plugins/jx-gitops/pull/835#discussion_r826644017

Reply:
Fixed.